charts/stable/docker-compose/values.yaml

```
@@ -60,7 +41,24 @@ workload:
           env:
             DOCKER_TLS_CERTDIR: /certs
             COMPOSE_FILE: ""
-
+          lifecycle:
+            postStart:
+              type: exec
+              command:
+                - /bin/bash
+                - -c
+                - |
+                  if [ "${COMPOSE_FILE}" ]; then
```

```suggestion
                  if [ -f "${COMPOSE_FILE}" ]; then
```

Isn't this needed here? My bash is a bit rusty

---

They're testing if the variable is set, not that the file exists. The `echo "COMPOSE_FILE not set, not auto-loading compose-file"` in the `else` block mentions this.